docs/sphinx/requirements.txt

```diff
@@ -0,0 +1,3 @@
+sphinx==3.0.3
+recommonmark==0.6.0
+-e git+git://github.com/pytorch/pytorch_sphinx_theme.git#egg=pytorch_sphinx_theme
```

Comment:
The pip install of sphinx didn't cut it... 
```
(ts_docs) /Users/markhama/Development/a2-serve/docs/sphinx markhama@f01898b363ea% make html                             
/bin/sh: sphinx-build: command not found
```
So I did a `conda install sphinx=3`
And that got me past that problem... 
I also had to install:

```
pip install sphinxcontrib-katex torch
```
And I still ran into issues with `make html`... we need at least a stub index.rst in there like...

```
############
TorchServe
############

.. toctree::
   :maxdepth: 1
   :glob:

   *
```
Even so there are a lot of warnings. 
But I think if you update the requirements and validate you can do a fresh env and get a build, then we're at a good starting point to layer on the rest.

Reply:
Thanks for the review

* Added `sphinxcontrib-katex` and `torch`. Note they will also be added as part of the github actions
* I was able to install sphinx by running `pip install -r requirements.txt` from the docs/sphinx folder. The github actions will similarly use `pip install`.
* The index.rst file will be added in [PR #290 (https://github.com/pytorch/serve/pull/290/files)
* The warnings are syntax suggestions in the md files; they aren't breaking, but something we can clean up later if desired

Reply:
Thanks for the review

* Added `sphinxcontrib-katex` and `torch`. Note they will also be added as part of the github actions
* I was able to install sphinx by running `pip install -r requirements.txt` from the docs/sphinx folder. The github actions will similarly use `pip install`.
* The index.rst file will be added in [PR #290 (https://github.com/pytorch/serve/pull/290/files)
* The warnings are syntax suggestions in the md files; they aren't breaking, but something we can clean up later if desired